### PR TITLE
Support UXP applications

### PR DESCRIPTION
**Pull request by g4jc**
This enables SSL Sleuth to work without error on Pale Moon and Iceweasel-UXP.

### CODE REVIEW

File: content/preferences.xul [hunk 1]
```diff
@@ -376,13 +376,6 @@
 							but WITHOUT ANY WARRANTY. See the GNU General Public License for more details.
 						</description>
 
-						<description value=""/>
-
-						<description class="ssleuth-pref-text">
-							Jdenticon 1.3.2 zlib license, jdenticon.com (c) Daniel Mester Pirttijärvi
```
Comment (rugk) [inline content/preferences.xul]: I doubt removing license disclaimers help…

File: install.rdf [hunk 2]
```diff
@@ -6,31 +6,39 @@
...
+                    <maxVersion>28.*</maxVersion>
                 </r:Description>
             </targetApplication>
 
             <localized>
                 <r:Description>
                     <locale>de-DE</locale>
                     <name>SSleuth</name>
-                    <description>How strong is your HTTPS connection? SSleuth ranks an established SSL/TLS connection and provides a brief summary of the SSL/TLS parameters. Lets users tweak cipher suites status.</description>
+                    <description>Wie stark ist Ihre HTTPS-Verbindung? SSleuth ordnet eine bestehende SSL / TLS-Verbindung ein und bietet eine kurze Zusammenfassung der SSL / TLS-Parameter. Ermöglicht Benutzern, den Status der Chiffriersätze zu optimieren.</description>
```
Comment (rugk) [inline install.rdf]: wtf, why translated?

**Review by rugk — COMMENTED** [review]
This changes a lot and I doubt all of that is related to the change.

**Comment by sibiantony** [comment]
@g4jc,  as said via mail, the current code is obsolete with webextensions. And I don't plan to support for PaleMoon. Sorry. 
You're however free to patch and upload to PaleMoon addons site. Keep the license, and use a support site of your own.

File: content/preferences.xul (hunk 1 above)
Comment (g4jc) [reply content/preferences.xul]: I didn't write that change. It was copied verbatim from the XPI as it appeared on [AMO](https://web.archive.org/web/20180730040927/https://addons.mozilla.org/en-US/firefox/addon/ssleuth/versions/). It appears JDenticon was completely removed in 0.5.3, if you think this was in error I will add it back, but it wasn't in the last published version?

File: install.rdf (hunk 2 above)
Comment (g4jc) [reply install.rdf]: de-DE was not properly localized, the main description remains in English.

**Pull request closed without merging**